packages/react-examples/src/react-components/Migrations/utils.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { Source } from '@storybook/addon-docs/blocks';
+import { makeStyles } from '@fluentui/react-make-styles';
+
+const codeComparisonUseStyles = makeStyles({
+  root: {
+    display: 'flex',
+    justifyContent: 'space-between',
+
+    '> *': {
+      width: '100%',
+    },
+
+    '> :not(:last-child)': {
+      marginRight: '5%',
+    },
+  },
+});
+
+export const CodeComparison = (props: { children: React.ReactElement[] }) => {
```

Comment:
nit: might be clearner

```suggestion
export const CodeComparison: React.FC = ({children}) => {
```

Reply:
`React.FC` has the danger of defining `children` as `React.Node`, allowing for anything to be passed.
That is not the kind of API I want `CodeComparison` to have.

What do you think instead about:
![image](https://user-images.githubusercontent.com/39736248/119006808-fa422480-b990-11eb-987a-cd5c779897a8.png)

Reply:
hey folks, sorry to chime in. @andrefcdias  pinged me regarding React.FC and whatnot. 

I prepared short explanation ( probably out of scope in this PR, but to get better general understanding )

![image](https://user-images.githubusercontent.com/1223799/119006908-12b23f00-b991-11eb-9210-4022060e5eb2.png)